tests/Set/Nextcloud27/Fixture/controller_use_session_annotation.php.inc

```diff
@@ -5,9 +5,18 @@ namespace Nextcloud\Rector\Test\Set\Nextcloud27\Fixture;
 class SomeController
 {
     /**
+     * @AnonRateLimit
+     * @ARateLimit
+     * @AuthorizedAdminSetting
+     * @BruteForceProtection
```

Comment:
```suggestion
     * @AnonRateThrottle(limit=10, period=100)
     * @UserRateThrottle(limit=20, period=200)
     * @AuthorizedAdminSetting(settings=OCA\NotesTutorial\Settings\NotesAdmin;OCA\NotesTutorial\Settings\NotesSubAdmin)
     * @BruteForceProtection(action=login)
```

I have this locally but cannot push it because test fails.
It appears that the Rector `AnnotationToAttributeRector` loses parameters in rector 1.2.X.
I think it is fixed in Rector 2. So this has to wait for #34